index.bs

```diff
@@ -2995,7 +2995,7 @@ Methods</h4>
 		{{AudioNode}} object.
 
 		<pre class=argumentdef for="AudioNode/connect(destinationNode, output, input)">
-			destinationNode: The <code>destination</code> parameter is the {{AudioNode}} to connect to. <span class="synchronous">If the <code>destination</code> parameter is an {{AudioNode}} that has been created using another {{AudioContext}}, an {{InvalidAccessError}} MUST be thrown</span>. That is, {{AudioNode}}s cannot be shared between {{AudioContext}}s.
+			destinationNode: The <code>destination</code> parameter is the {{AudioNode}} to connect to. <span class="synchronous">If the <code>destination</code> parameter is an {{AudioNode}} that has been created using another {{AudioContext}}, an {{InvalidAccessError}} MUST be thrown</span>. That is, {{AudioNode}}s cannot be shared between {{AudioContext}}s. Multiple {{AudioNode}} can be connected to the same {{AudioNode}}, this is described in the section [[#channel-up-mixing-and-down-mixing|Channel Upmixing and down mixing]].
```

Comment:
Two small nits:

1. `Multiple {{AudioNode}}s`
2. `this is described in [[#channel-up-mixing-and-down-mixing|Channel up-mixing and down-mixing]] section.`